readded everything (#104)
MrLongbottom/P10-Article: sections/appendix/taxonomies.tex

\subsubsection{Taxonomy}\label{subsec:appendix_taxonomy}

> Taxonomy is fundamentally different from the other metadata fields, in this paper.

Reviewer: ```suggestion
The taxonomy field is fundamentally different from the other metadata fields.
```

It is not fully observed with only roughly $25\%$ of documents having a taxonomy field.

> It is hierarchical, with each taxonomy containing a sequence of sub-taxonomies, such as: 'STEDER/Danmark/Nordjylland/Aalborg'.

Reviewer: ```suggestion
It is hierarchical, with each taxonomy containing a sequence of taxonomy entries, such as: 'STEDER/Danmark/Nordjylland/Aalborg'.
```

> It is hierarchical, with each taxonomy containing a sequence of sub-taxonomies, such as: 'STEDER/Danmark/Nordjylland/Aalborg'.

Author: ```suggestion
It is hierarchical, with each taxonomy containing a sequence of taxonomy entries, such as: 'STEDER/Danmark/Nordjylland/Aalborg'.
Taxonomy sequences can have varying length, ranging from $1$ to $5$.
```

> It is also possible for each document to have multiple taxonomy sequences.

Reviewer 2: ```suggestion
It is also possible for each document to have multiple taxonomy sequences, such as having both "STEDER" and "EMNER" in two separate taxonomies.
```

> Like with authors, we remove any sub-taxonomy that is used in less than $14$ taxonomy sequences ($0.01\%$ of the number of documents).

Reviewer: ```suggestion
Like with authors, we remove any taxonomy entries that are used in less than $14$ taxonomy sequences (${\sim}0.01\%$ of the number of documents).
```

Out of $1135$ sub-taxonomies, $779$ are removed during this preprocessing.
A subset of the taxonomies and the number of articles they appear in can be seen in \autoref{tab:taxonomy_table}.

> The taxonomy tree has the following sizes: $1$, $4$, $32$, $90$, and $69.192$.
> The numbers $4$ and $32$ are based on the number of unique taxonomy fields observed in the dataset.
> The numbers $1$, $90$, and $69.192$ are the root, the number of topics, and the number of words, respectively.

Author: these sizes are based on the pachinko dag not the taxonomy tree, so they need to be updated
```suggestion
The taxonomy tree has the following sizes: $1$, $4$, $32$, $90$, and $69.192$.
The numbers $4$ and $32$ are based on the number of unique taxonomy fields observed in the dataset.
The numbers $1$, $90$, and $69.192$ are the root, the number of topics, and the number of words, respectively.
\todo{fix sizes to fit with taxonomy tree instead of pachinko dag. also move this to main paper so we know dag sizes. Tree sizes: 4, 32, 80, 99, 290}
```

For the labels in \autoref{tab:taxonomy_table}, the counts show the labels' sizes.
It is worth mentioning that, while the counts of the labels are directly linked to the number of documents in the whole dataset, they are also connected to the size of the previous layer's taxonomy.

> For example, the label 'Danmark' is in the layer below the 'STEDER' label, where the 26145 documents of 'Danmark' is a subset of the 29535 documents of 'STEDER'.

Reviewer 2: ```suggestion
For example, the label 'Danmark' is in the layer below the 'STEDER' label, where the 26145 documents of 'Danmark' are a subset of the 29535 documents of 'STEDER'.
```

\todo{fix sizes to fit with taxonomy tree instead of pachinko dag. also move this to main paper so we know dag sizes. Tree sizes: 4, 32, 80, 99, 290}

For the labels in \autoref{tab:taxonomy_table}, the counts show the labels' sizes.
It is worth mentioning that, while the counts of the labels are directly linked to the number of documents in the whole dataset, they are also connected to the size of the previous layer's taxonomy.
For example, the label 'Danmark' is in the layer below the 'STEDER' label, where the 26145 documents of 'Danmark' are a subset of the 29535 documents of 'STEDER'.

There is also a much larger number of 'STEDER' (places) documents compared to 'EMNER' (subjects) documents, respectively 29535 compared to 5449.
In \autoref{tab:taxonomy_table}, the first label on a lower layer that comes from 'EMNER' is 'Sport' with 408 documents.
This indicates that the subject taxonomies, in general, are much smaller in size.
The \gls{pam} will then be mostly influenced by this location information compared to actual topical labels.

The labels removed during preprocessing appear, as expected, to be places or subjects that have been written about just once or a few times.

> We could have combined them into a 'misc' taxonomy as we did for the category and author metadata, but decided against it because it would just be a large collection of mostly random documents.

Author: ```suggestion
We could have combined them into a 'misc' taxonomy as we did for the category and author metadata, but decided against it because it would just be a large collection of mostly random documents.
The taxonomy sequences that had some taxonomy entries removed, also still contain other taxonomy entries so they are not entirely without information as with the 'misc' author and category.
```
